interceptors/retry/examples_test.go

```diff
@@ -88,3 +91,39 @@ func ExampleWithPerRetryTimeout() {
 		retry.WithMax(3),
 		retry.WithPerRetryTimeout(1*time.Second))
 }
+
+// scale duration by a factor.
+func scaleDuration(d time.Duration, factor float64) time.Duration {
+	return time.Duration(float64(d) * factor)
+}
+
+func TestJitterUp(t *testing.T) {
+	// arguments to jitterup.
```

Comment:
```suggestion
	// Arguments for jitterup.
```

Comments should be valid English sentences! (: (Capital letter on front) - same above and below ;p

Reply:
My bad!